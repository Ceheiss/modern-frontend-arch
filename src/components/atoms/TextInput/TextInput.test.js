import React from "react";
import { screen, render } from "@testing-library/react";

import TextInput from "./TextInput";

describe("TextInput", () => {
  it("should show container", () => {
    const { container } = render(<TextInput />);
    expect(container).toMatchSnapshot();
  });
});
