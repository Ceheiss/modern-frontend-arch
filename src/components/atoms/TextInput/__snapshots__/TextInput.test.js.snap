// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div>
  <h1>
    Hello from TextInput
  </h1>
</div>
`;

exports[`TextInput should show container 1`] = `
<div>
  <h1>
    Hello from TextInput
  </h1>
</div>
`;
